padinfo/view/pantheon.py

```diff
@@ -16,10 +17,14 @@
 if TYPE_CHECKING:
     from dadguide.models.monster_model import MonsterModel
 
+MAX_MONS_TO_SHOW = 20
+MAT_TYPES = ['Evolve', 'Awoken', 'Enhance', 'Vendor']
+NIL_ATT = 'Nil'
```

Comment:
How about using the enum values instead of string names?

IDEs know enums better than strings. If you misspell "Enhance" ("Enhanced"), you won't get an error. If you misspell `MonsterType.Enhance`, you will.

Reply:
No, the enum is located in another cog, and cross-cog imports are impossible. This is extremely deliberate.